dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/manager/DefaultExecutorRepository.java

```diff
@@ -354,10 +346,11 @@ public ScheduledExecutorService getSharedScheduledExecutor() {
     }
 
     private ExecutorService createExecutor(URL url) {
-        ExecutorService executorService = (ExecutorService) ExtensionLoader.getExtensionLoader(ThreadPool.class).getAdaptiveExtension().getExecutor(url);
+        ExecutorService executorService = (ExecutorService) extensionAccessor.getExtensionLoader(ThreadPool.class).getAdaptiveExtension().getExecutor(url);
 
         DataStore dataStore = ExtensionLoader.getExtensionLoader(DataStore.class).getDefaultExtension();
```

Comment:
```extensionAccessor```'s scope might not be default application, ```executorService``` created by ```extensionAccessor``` will be destroyed with ```extensionAccessor```'s application by DefaultApplicationDeployer.destroyExecutorRepository().
```dataStore```'s scope might be same as  ```executorService``` to avoid multiple applications problem.